kratos/sources/variables.cpp

```diff
@@ -298,10 +298,10 @@ KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY_COMPONENT_GRADIENT)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY_X_GRADIENT)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY_Y_GRADIENT)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY_Z_GRADIENT)
-KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(DISPLACEMENT)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(REACTION)
-KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(VELOCITY)
 KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS(ACCELERATION)
+KRATOS_CREATE_3D_VARIABLE_WITH_COMPONENTS_WITH_TIME_DERIVATIVE(VELOCITY, ACCELERATION)
```

Comment:
can you please do it for `ROTATION` right away? This is also used in the BDF scheme in structural

Reply:
i did it, I will commit now, testing...